data/io.elementary.files.appdata.xml.in.in

```diff
@@ -56,6 +56,23 @@
     <content_attribute id="money-gambling">none</content_attribute>
   </content_rating>
   <releases>
+    <release version="6.0.1" date="2021-07-15" urgency="medium">
+      <description>
+        <p>Fixes:</p>
+        <ul>
+          <li>Fix truncation of slot if reclicked in Column View</li>
```

Comment:
What does "slot" mean in this context?

Reply:
I guess "column" would be a better word.  A slot in the context of Files is the widget that displays a directory listing.